### PR TITLE
Carthage curator moderation actions

### PR DESCRIPTION
**Pull request by zeeshanakram3**
implements #3870

Based on #3921 so expects it to be merged first

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202132419573087/1202500354271984) by [Unito](https://www.unito.io)

### CODE REVIEW

File: tests/network-tests/src/fixtures/content/createCuratorGroupFixture.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+          ? { DeleteVideoAssets: permissionToDeleteNftAssets }
+          : a === 'HideChannel'
+          ? { HideChannel: null }
+          : a === 'HideVideo'
+          ? { HideVideo: null }
+          : a === 'DeleteNonVideoChannelAssets'
+          ? { DeleteNonVideoChannelAssets: null }
+          : a === 'UpdateChannelNftLimits'
+          ? { UpdateChannelNftLimits: null }
+          : { ChangeChannelFeatureStatus: { CreatorCashout: null } }
```
Comment (ondratra) [inline tests/network-tests/src/fixtures/content/createCuratorGroupFixture.ts]: This part is very hard to read. Let's replace this ternary operators chain by a helper function that will return proper value(s).

**Review by ondratra — CHANGES_REQUESTED** [review]
I found one spot that could use a better code style, but otherwise this PR looks good.

**Review by ondratra — APPROVED** [review]
LGTM

**Comment by mnaamani** [comment]
small conflict fix needed.

**Pull request merged**